Controller/ImagineController.php

```diff
@@ -155,4 +174,14 @@ private function createRedirectResponse(\Closure $url, string $path, string $fil
             ]), 0, $exception);
         }
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return bool
+     */
+    private function isSupportWebp(Request $request)
```

Comment:
You can add return type hint `: bool` and remove method annotation (it's useless)